op-chain-ops/genesis/config.go

```diff
@@ -174,5 +181,8 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block, proxyL1Standar
 		// TODO: this should be set to the MintManager
 		"_owner": common.Address{},
 	}
+	storage["ProxyAdmin"] = state.StorageValues{
+		"owner": l2Addrs.ProxyAdmin,
```

Comment:
Is this making the owner of the proxy admin be itself? Shouldn't the owner be `ProxyAdminOwner`?

Reply:
I can update the variable naming

Reply:
I've updated the variable name, it was always the owner